docs/user/learner.rst

```diff
@@ -4,24 +4,27 @@ Learner
 Accessing Kolibri
 -----------------
 
-To log in to **Kolibri** follow these steps:
+To sign in to **Kolibri** and start learning follow these steps:
 
-#. Click the **Log In** icon in the main menu (left or bottom) to open the login window.
+#. Click the **Sign in** icon either in the upper right corner, or in the main menu (left or bottom) to open the sign-in page.
```

Comment:
I think we should specify that you would only ever have to sign in using the menus if you decided to "Access as Guest" - currently, the default behavior of the app when no one is signed on is to take you to the `user` plugin.